path_config.go

```diff
@@ -17,10 +17,18 @@ func (b *backend) pathConfig() *framework.Path {
 	return &framework.Path{
 		Pattern: "config",
 		Fields: map[string]*framework.FieldSchema{
-			"certificates": {
-				Required:    true,
-				Type:        framework.TypeStringSlice,
-				Description: "The PEM-format CA certificates.",
+			"identity_ca_certificates": {
+				Required:     true,
+				Type:         framework.TypeStringSlice,
+				DisplayName:  "Identity CA Certificates",
+				DisplayValue: `-----BEGIN CERTIFICATE----- ... -----END CERTIFICATE-----`,
+				Description:  "The PEM-format CA certificates that are required to have issued the instance certificates presented for logging in.",
+			},
+			"pcf_api_trusted_certificates": {
```

Comment:
I added this field because I found that PCF often uses self-signed certificates. I didn't want to encourage users to use an insecure TLS connection, so instead (on Jim's idea) added the ability to trust a given API cert, and added documentation on how to retrieve it.